db/mysql/ops.go

```diff
@@ -0,0 +1,264 @@
+package mysql
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	domain "github.com/Hammad887/chat-app/models"
```

Comment:
arrange imports

Reply:
Fixed. Used goimports to arrange the imports